apps/client/src/features/rundown/event-block/composite/EventBlockReporter.tsx

```diff
@@ -0,0 +1,93 @@
+import { Tooltip } from '@chakra-ui/react';
+import { OntimeEvent, ReportData } from 'ontime-types';
+import { isPlaybackActive, MILLIS_PER_MINUTE, MILLIS_PER_SECOND } from 'ontime-utils';
+
+import { usePlayback, useTimeUntil } from '../../../../common/hooks/useSocket';
+import useReport from '../../../../common/hooks-query/useReport';
+import useRundown from '../../../../common/hooks-query/useRundown';
+import { cx } from '../../../../common/utils/styleUtils';
+import { formatDuration, getTimeToStart } from '../../../../common/utils/time';
+import { tooltipDelayFast } from '../../../../ontimeConfig';
+
+import style from './EventBlockReporter.module.scss';
+
+interface EventBlockReporterProps {
```

Comment:
cleanup: looks like this is used in `EventBlockChip`, so it should be called EventBlockChipProps and be next to the function that uses it